kotlin/viewshed-location/README.metadata.json

```
@@ -0,0 +1,29 @@
+{
+    "category": "Analysis",
+    "description": "A viewshed shows the visible and obstructed areas from an observer's vantage point. This sample demonstrates all of the configurable properties of a viewshed.",
```

This description should match the summary from the readme
```suggestion
    "description": "Perform a viewshed analysis from a defined vantage point.",
```